src/Listing/index.tsx

```diff
@@ -353,9 +353,9 @@ const Listing = ({
   const emptyRows = limit - Math.min(limit, totalRows - currentPage * limit);
 
   const tableMaxHeight = (): string => {
-    return innerScrollDisabled
-      ? '100%'
-      : `calc(100vh - ${tableTopOffset}px - 25px)`;
+    return !innerScrollDisabled
+      ? `calc(100vh - ${tableTopOffset}px - 100px)`
+      : '100%';
```

Comment:
```suggestion
    if (innerScrollDisabled) {
      return '100%';
    }
    
    return `calc(100vh - ${tableTopOffset}px - 100px)`;
```

I don't remember why we have to retract either 25px or 100px, but we should document it via a variable.